readme.md

```diff
@@ -9,22 +9,20 @@ expect('.toggle-pane').to.not.be.visible()
 
 ## What sorts of assertions can we make?
 
-All assertions start with a css-selector, for example:
+All assertions start with a [WebdriverIO-compatible selector](http://webdriver.io/guide/usage/selectors.html), for example:
 
-- `expect('.list')`
-- `expect('div > h1')`
-- `expect('a[href=http://google.com]')`
+- `expect('.list')` (CSS selector)
+- `expect('a[href=http://google.com]')` (CSS Selector)
+- `expect('//BODY/DIV[6]/DIV[1]')` (XPath selector)
+- `expect('a*=Save')` (Text selector)
 
-Then we add the dom flag, like so:
+Then, we can add our assertion to the chain.
 
-- `expect(selector).dom`
```

Comment:
So this actually isn't required, right?  (Just a copy/paste oversight from `chai-webdriver` maybe?)